automation/bootstrap.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+PKGS="epel-release git gcc python-devel lvm2 docker"
```

Comment:
its nicer to use shell lists , e.g.:

    PKGS=(aaa bbb ccc)
    yum install -y "${PKGS[@]}"

etc...

Reply:
+1